core/sql/comexe/ComTdbExeUtil.cpp

```diff
@@ -1302,6 +1302,7 @@ void ComTdbExeUtilFastDelete::displayContents(Space * space,
 ComTdbExeUtilHiveTruncate::ComTdbExeUtilHiveTruncate(
      char * tableName,
      ULng32 tableNameLen,
+     char * hiveTableName,
```

Comment:
Could you explain what you mean by "bound"? This is just added here to include the table name in the TDB, so we have access to it at runtime. Maybe you are saying that we could use a host variable for the name instead? Yes, that would be doable, but it's not currently done for the truncate command and it probably wouldn't be easy for most other commands such as load, etc. In this particular case, I'm just adding the Hive table name, the SQL statement uses the ANSI SQL name, e.g. HIVE.HIVE.T, and that name is also stored in the TDB.